### ISSUE TITLE
No map details actually showing

### ISSUE THREAD

**Issue by oakleaf** [hansemannn/titanium-google-maps#102]
Hello, awesome module. I have pasted everything into my app.js, followed the whole guide, pasted our API Key (Which works in the browser, I created a new one and restricted it to this app).

The mapview shows up, polylines too. But nothing shows on the actual map. I mean, there's no map. Just a beige background with polylines.

Since no error messages are showing, I have no clue how to debug.

I use the latest version of the module (3.8) together with latest Ti SDK 6.1.2 and with iOS SDK 10.3.

Any idea how I can debug this?

Thanks!

**Comment by hansemannn** [comment]
You have to specify a valid API-key that is linked to your app. After whitelisting your app, it will render correctly.

**Comment by oakleaf** [comment]
Sorry for keep on asking this, but I have done just that. In the google api console, I have created a new API key for maps, where I have specified my app ID (com.mappy.app), and I have also, in the Apple developer-portal created that appId. I have waited more than 5 minutes until the changes should apply in the google api console, as it says I should do. Key restriction is also set to iOS apps.

This doesnt work on the device, nor the simulator. Any other tip?

<img width="653" alt="skarmavbild 2017-09-02 kl 11 07 26" src="https://user-images.githubusercontent.com/441910/29994422-13e1e30c-8fcf-11e7-8069-ba6e31e608d5.png">
